File: simulx-R/test_simulx.R
library(gridExtra)

p <- c(tau1 = 1, 
       tau2 = .2) 

out <- list(name=c('f1', 'f2','f3'), time=seq(0,5,by=.001))

res <- simulx(model     = 'dde2.txt', 
              parameter = p, 
              output    = out)

plot(res$f1,type="l",col="blue",ylim=c(0,200))
lines(res$f2,col="green")
lines(res$f3,col="red")
axis(4,at=seq(0,200,by=20))
grid()

File: simulx-matlab/test_simulx.m
p=struct;
p.name={'tau1','tau2'};
p.value=[1,.2];

f=struct;
f.name={'f1','f2','f3'};
f.time=linspace(0,5,100);

res = simulx( 'model','dde2.txt', 'parameter',p, 'output',f);

% plotmlx documented in https://team.inria.fr/popix/files/2014/08/Mlxm112.pdf
opts.output={{'f1','f2','f3'}};  % bc. potentially >1 panel
plotmlx(res,'option',opts);

% Wasting one's short life reshaping between cells and matrices to make
% mathworks rich
% plot(f.time,cell2mat([res{1}.value; res{2}.value; res{3}.value]));

